bin/build-release-ccd-definition.sh

```diff
@@ -3,6 +3,8 @@
 set -eu
 
 environment=${1:-prod}
+declare -a unshutterlist=["prod","staging"]
```

Comment:
this line I think should define list of shuttered env, to make it simple. And normally this will be empty. but we will add env name here to shutter whenever needed